### ISSUE TITLE
Merge `min_config_state_channel_subscriptions.feature` and `min_config_state_channel.feature`

### ISSUE THREAD

**Issue by meaksh** [uyuni-project/uyuni#7963]
Issue created by automation script. Original author: @Bischoff Date: 2018-09-13T09:19:55Z
Source issue: https://github.com/SUSE/spacewalk/issues/5798

---
There is a chance there is duplicate code between:

* `min_config_state_channel_subscriptions.feature` (https://github.com/uyuni-project/uyuni/commit/0792476eab30644acadb79a5736f5388a4db8b7f)
* `min_config_state_channel.feature`

so we might be able to spare execution time by merging them.

Even without factorizing, the following two problems need be addressed:

* the name of both features is very similar, it is confusing
* name of scenarios is duplicate, which prevents using `cucumber -n "scenario name"`

Low priority.

**Comment by meaksh** [comment]
Comment created by automation script. Original author: @Bischoff. Date: 2018-09-13T09:21:51Z

---
Explanations from @mcalmer : 
> features/min_state_config_channel.feature test a problem we found.
> You can create two channels with the same "name". Only the "label" needs to be 
> different. This is ok as long as no UI show just the name which was the case.
>
> The bug in the past was: 
>
> Some UIs show two time "My State Channel" no idea what configfiles are behind 
> this. Some other UIs just showed "My State Channel" just 1 time. So you lost info.
>
> This test check these UIs and if the enhancement to append the label to the 
> name works.
>
> To implement this I just re-used an existing test, but it is not the same.
>
> The only thing you can do is, to join them together in one feature file.

**Comment by meaksh** [comment]
Comment created by automation script. Original author: @ktsamis. Date: 2021-01-26T10:59:12Z

---
Barely had time to look at this, putting back in inbox now that I'm unblocked. If I have the time I will take it again but I will be pretty busy the next few days

**Comment by meaksh** [comment]
Comment created by automation script. Original author: @srbarrios. Date: 2022-01-13T16:49:07Z

---
@ktsamis one year later...

**Comment by nodeg** [comment]
Moving back to low priority.
